frontend/app/assets/stylesheets/spree/frontend/components-custom/inputs.scss

```diff
@@ -19,7 +18,7 @@
     font-size: font-px-to-rem(14px);
   }
   &:focus {
-    border-color: theme-color('primary');
```

Comment:
Please, use previous value and add the variable here instead frontend/app/assets/stylesheets/spree/frontend/variables/bootstrap-overrides.scss#4